cves/CVE-2007-5000.yml

```diff
@@ -140,12 +135,10 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: There were some files that had a name change during development.
   events:
-    - name:
-      date:
-    - name:
-      date:
+    - name: mod_imap renamed to mod_imagemap
+      date: April 6, 2005
```

Comment:
Format of date might have to be yyyy-mm-dd

Reply:
Thank you. I'll change it.